content/sigs/cloud-native/pluggable-storage.adoc

```diff
@@ -29,73 +29,107 @@ link:/sigs/cloud-native[Cloud Native SIG Page].
 
-| Build logs
-| In progress
-| jep:207[], jep:210[], jep:212[]
-|
+| plugin:artifact-manager-s3[Artifact Manager on S3],
+  plugin:azure-artifact-manager[Azure Artifact Manager],
+  link:/doc/developer/extensions/jenkins-core/#artifactmanagerfactory[<More implementations>]
 
-| System logs
-| Available
-| N/A
+| **Credentials** +
+  **(Available)**
+| Completed before the JEP process was introduced.
+
+link:https://www.jenkins.io/doc/developer/extensions/credentials/#credentialsprovider[Extension point]
+| plugin:kubernetes-credentials-provider[Kubernetes Credentials Provider],
+  plugin:aws-secrets-manager-credentials-provider[AWS Secrets Manager Credentials Provider],
+  https://www.jenkins.io/doc/developer/extensions/credentials/#credentialsprovider[<More implementations>]
+
+| Build logs +
+  (Preview / Paused)
+| The story is completed for Pipeline Log Storage, only Jenkins Pipeline job types are supported.
+  Related JEP: jep:210[].
+
+  Jenkins core APIs and reference implementations have not been merged/released yet,
+  but prototypes are available for evaluation.
+  Related JEPs: jep:207[], jep:212[]
+a| Pipeline logging:
+
+* https://github.com/jenkinsci/pipeline-cloudwatch-logs-plugin[AWS CloudWatch]
+* https://github.com/SAP/elasticsearch-logs-plugin[Elasticsearch]
+
+Jenkins core:
+
+* https://github.com/jenkinsci/external-logging-api-plugin[External Logging API]
+* https://github.com/jenkinsci/external-logging-elasticsearch-plugin[Elasticsearch]
+
+| **System logs** +
+  **(Available)**
 | Jenkins supports custom log appenders using standard `java.util.logging`
   link:https://tutorials.jenkov.com/java-logging/configuration.html[configuration options].
+| plugin:syslog-logger[Syslog logger],
+  non-Jenkins implementation
 
-
-| Task logs
-| Not started
-|
+| Task logs +
+  (Not started)
 | Storage of system logs and various tasks (e.g. agent connection or SCM polling)
+| N/A
 
-| Configurations
-| In progress
-| jep:213[]
-|
-
-| Test results
-| In progress
-| Coming soon
-|
-
-| Code coverage results
-| In exploration
-| TBD
-|
-
-| Runs
-| Not started
-|
-| Storage of full run records in an external database
-
-| Jobs
-| Not started
-|
+| Configurations +
+  (Paused)
+| Largely replaced by the plugin:configuration-as-code[Configuration as Code] plugin
+  which allows storing Jenkins configurations in SCM or other locations without a database.
+
+  Related JEPs: jep:213[]
+| N/A
+
+| **Test Results** + 
+  **(Preview)**
+| Foundation API is integrated into plugin:junit[JUnit Plugin 1.32].
+  Remaining changes are tracked as link:https://github.com/jenkinsci/junit-plugin/issues/142[junit #142]
+| https://github.com/timja/jenkins-junit-postgresql-plugin[JUnit PostgreSQL Plugin]
+
+| Code Coverage results +
+  (Not started)
+| Planned only for plugins based on plugin:code-coverage-api[Code Coverage API] which unifies the storage implementation.
+  See _Runs_ for other coverage report types.
+| N/A
+
+| Runs +
```

Comment:
```suggestion
| Builds +
```

Reply:
```suggestion
| Builds/Runs +
```

We still need to fixate the glossary as we discussed with @megathaum a while ago

Reply:
Will incorporate changes. Right now Builds are referenced in https://www.jenkins.io/doc/book/glossary/